pkg/config/system_probe.go

```diff
@@ -0,0 +1,99 @@
+package config
+
+import (
+	"strings"
+)
+
+const (
+	spNS  = "system_probe_config"
+	netNS = "network_config"
+
+	defaultConnsMessageBatchSize = 600
+
+	// defaultSystemProbeBPFDir is the default path for eBPF programs
+	defaultSystemProbeBPFDir = "/opt/datadog-agent/embedded/share/system-probe/ebpf"
+
+	// defaultRuntimeCompilerOutputDir is the default path for output from the system-probe runtime compiler
+	defaultRuntimeCompilerOutputDir = "/var/tmp/datadog-agent/system-probe/build"
+
+	defaultOffsetThreshold = 400
+)
+
+func setupSystemProbe(cfg Config) {
+	// settings for system-probe in general
+	cfg.BindEnvAndSetDefault(join(spNS, "enabled"), false, "DD_SYSTEM_PROBE_ENABLED")
+	cfg.BindEnvAndSetDefault(join(spNS, "external"), false, "DD_SYSTEM_PROBE_EXTERNAL")
+
+	cfg.BindEnvAndSetDefault(join(spNS, "sysprobe_socket"), defaultSystemProbeAddress, "DD_SYSPROBE_SOCKET")
+	cfg.BindEnvAndSetDefault(join(spNS, "max_conns_per_message"), defaultConnsMessageBatchSize)
+
+	cfg.BindEnvAndSetDefault(join(spNS, "log_file"), defaultSystemProbeLogFilePath)
+	cfg.BindEnvAndSetDefault(join(spNS, "log_level"), "info", "DD_LOG_LEVEL", "LOG_LEVEL")
+	cfg.BindEnvAndSetDefault(join(spNS, "debug_port"), 0)
+
+	cfg.BindEnvAndSetDefault(join(spNS, "dogstatsd_host"), "127.0.0.1")
+	cfg.BindEnvAndSetDefault(join(spNS, "dogstatsd_port"), 8125)
+
+	cfg.BindEnvAndSetDefault(join(spNS, "profiling.enabled"), false, "DD_SYSTEM_PROBE_PROFILING_ENABLED")
+	cfg.BindEnvAndSetDefault(join(spNS, "profiling.site"), DefaultSite, "DD_SYSTEM_PROBE_PROFILING_SITE", "DD_SITE")
+	cfg.BindEnvAndSetDefault(join(spNS, "profiling.profile_dd_url"), "", "DD_SYSTEM_PROBE_PROFILING_DD_URL", "DD_APM_PROFILING_DD_URL")
```

Comment:
`DD_SYSTEM_PROBE_PROFILING_DD_URL` seems to be a new env var and does not match `profiling.profile_dd_url`. Is it expected?

Reply:
I was matching the format of `DD_APM_PROFILING_DD_URL` but specific to system-probe. Happy to change it, if we think something else is better.